commit-based-cipm/fi-tests/cipm.consistency.fitests.repositorytests/src/cipm/consistency/fitests/repositorytests/CWARepoTest.java

```diff
@@ -43,8 +43,18 @@ protected URI getRepoURI() {
 	@Override
 	protected Collection<AbstractJaMoPPParserSimilarityTestFactory> getTestFactories() {
 		var res = new ArrayList<AbstractJaMoPPParserSimilarityTestFactory>();
-		res.add(new EAllContentSimilarityTestFactory(this.getSCC()));
+		// TODO Determine whether the order of Resource contents should matter
+		// Especially important due to synthetic elements from TrivialRecovery
+		// Use ModelComparisonTestFactory(boolean) to set it (true by default)
+		res.add(new ModelComparisonTestFactory());
 		res.forEach((tf) -> tf.setExpectedSimilarityResultProvider(getExpectedSimilarityResultProviderForCommits()));
 		return res;
 	}
+
+	@Override
+	protected RepoParserTestOptions initResourceTestOptions() {
+		var opts = super.initResourceTestOptions();
+		opts.setShouldDeleteRepositoryClones(true);
+		return opts;
+	}
 }
```

Comment:
This is a code duplication, which should be avoided. The same code is also in the `TeammatesRepoTest` class.